src/pages/reports/Reports.tsx

```diff
@@ -1,16 +1,108 @@
-import Strings from "../../utils/Strings";
-import LayoutCard from "../layouts/LayoutCard";
-const Reports = () => {
-    return (
-        <LayoutCard
-            title={Strings.reports}
-            isLoading={false}
-            content={
-                <div className="flex flex-col">
-                   
-                </div>
-            } />
-    );
+import React, { useState } from 'react';
+import { Button, Collapse, DatePicker, Form, Table, Row, Col } from 'antd';
+import LayoutCard from '../layouts/LayoutCard';
+import SectionElement from '../components/SectionElement';
+import Strings from '../../utils/Strings';
+import { usePostReportsMutation } from '../../services/reportsService';
+import { handleErrorNotification } from '../../utils/Notifications';
+import { Reports } from '../../data/reports/report';
+const { Panel } = Collapse;
+
+interface Report {
+	name: string;
+	lastname: string;
+	primaryContact: string;
+	appointment: string;
 }
 
-export default Reports;
\ No newline at end of file
+const ReportsComponent: React.FC = () => {
+	const [started_at, setStarted_at] = useState<string | null>(null);
```

Comment:
in programming side its not a good practice to use snake_case, please use camelCase

Reply:
I have already made the changes to camelCase

Reply:
Thanks